scripts/build-and-test.sh

```diff
@@ -54,8 +60,7 @@ if [[ $VARIANT == "coverage" ]]; then
   jobs
   # Kill it gracefully
   kill -INT %1
-  sleep 1
-  kill -INT %1 || echo "Dead already"
+  wait
```

Comment:
fwiw, since I added in the 5 second sleep after wstest, I haven't noticed any further issues.